http/src/test/scala/org/http4s/blaze/http/http2/WriteControllerImplSpec.scala

```diff
@@ -0,0 +1,149 @@
+package org.http4s.blaze.http.http2
+
+import java.nio.ByteBuffer
+
+import org.http4s.blaze.http.http2.mocks.MockTools
+import org.http4s.blaze.pipeline.TailStage
+import org.specs2.mutable.Specification
+
+import scala.collection.mutable
+import scala.concurrent.{Future, Promise}
+
+class WriteControllerImplSpec extends Specification {
+
+  private def mockData(i: Int): ByteBuffer =
+    ByteBuffer.wrap {
+    (0 until (i + 1)).map(_.toByte).toArray
```

Comment:
I guess my point here was `to` means you don't need to `+ 1`, unless you think `+ 1` is instructive.

Reply:
Oh, I see. I don't necessarily think the +1 is important so I'll change it.